### PR TITLE
Add release note for 2201.5.5

### PR DESCRIPTION
**Pull request by NipunaMadhushan**
## Purpose
$subject

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+redirect_from:
+- /downloads/swan-lake-release-notes/2201.5.5
+- /downloads/swan-lake-release-notes/2201.5.5/
+- /downloads/swan-lake-release-notes/2201.5.5-swan-lake/
+- /downloads/swan-lake-release-notes/2201.5.5-swan-lake
+---
+
+## Overview of Ballerina Swan Lake Update 5 (2201.5.5)
+
+<em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the <ADD_THE_LIST_OF_COMPONENTS>.</em>
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md:15]:
```suggestion
<em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the <ADD_THE_LIST_OF_COMPONENTS>.</em>
```
Need to fill the `<ADD_THE_LIST_OF_COMPONENTS>`.

File: downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Update your current Ballerina installation directly to 2201.5.5 by using the [Ballerina Update Tool](/learn/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist pull 2201.5.5` to update to this latest distribution.
+
+## Install Ballerina
+
+If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.
+
+<!-- ADD ONLY THE APPLICABLE SECTIONS FROM THE BELOW -->
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md:28]:
```suggestion
<!-- ADD ONLY THE APPLICABLE SECTIONS FROM THE BELOW -->
```
Add only the sections that are applicable for this release from the template.

File: downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+To view bug fixes, see the [GitHub milestone for Swan Lake Update 8 (<RELEASE_VERSION>)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+label%3AType%2FBug+is%3Aclosed+milestone%3A<RELEASE_VERSION>).
+
+## Developer tools updates
+
+### Bug fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Update 8 (<RELEASE_VERSION>) of the repositories below.
+
+- [Language server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FLanguageServer+milestone%3A<RELEASE_VERSION>+is%3Aclosed+label%3AType%2FBug+)
+- [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Swan+Lake+<RELEASE_VERSION>%22+is%3Aclosed)
```
Comment (lnash94) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md:55]:
```suggestion
- [OpenAPI](https://github.com/ballerina-platform/ballerina-library/issues?q=is%3Aissue+label%3Amodule%2Fopenapi-tools+milestone%3A2201.5.5+is%3Aclosed)
```

File: downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+redirect_from:
+- /downloads/swan-lake-release-notes/2201.5.5
+- /downloads/swan-lake-release-notes/2201.5.5/
+- /downloads/swan-lake-release-notes/2201.5.5-swan-lake/
+- /downloads/swan-lake-release-notes/2201.5.5-swan-lake
+---
+
+## Overview of Ballerina Swan Lake Update 5 (2201.5.5)
+
+<em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the tooling.</em>
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md:15]:
```suggestion
<em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to developer tools.</em>
```

**Review by lnash94 — DISMISSED** [review]

**Review by sm1990 — APPROVED** [review]

**Pull request merged**